bugwarrior/services/jira.py

```diff
@@ -188,8 +188,11 @@ def __get_sprints(self):
             for key in self.origin['sprint_field_names']
         ], [])
         for sprint in sprints:
-            # Parse this big ugly string.
-            yield _parse_sprint_string(sprint)
+            if isinstance(sprint, dict):
+                yield sprint
+            else:
```

Comment:
I'd like to add a comment here like `else:  # Old version of python-jira without sprint support.`

Reply:
This is actually not a new feature of python-jira but Atlassian seems to have fixed it in Jira itself. I left the old code as a backup as I don't know if it was fixed everywhere.

Reply:
I see. I still think some comment indicating that the else-clause is for backwards compatibility would be nice.

Reply:
Agreed, I added a comment.